modules_basic/like.js

```diff
@@ -43,8 +43,8 @@ exports.create = function (api) {
     if (votes.length === 0) return
 
     digs.textContent = votes.length > 4
-      ? votes.length + ' 🗸'
-      : Array(votes.length).fill('🗸').join('')
+      ? votes.length + ' \u2713'
+      : Array(votes.length).fill('\u2713').join('')
```

Comment:
This could have been a different pull request - only just saw it.
I've fixed and published a solution alread !